images/entrypoint.sh

```diff
@@ -403,6 +408,7 @@ EOF
         {
           $CNI_VERSION_STRING
           "name": "$MASTER_PLUGIN_NET_NAME",
+          "confDir": "$MULTUS_CONF_DIR",
```

Comment:
I'd prefer that we have it like below for `$NESTED_CAPABILITIES_STRING` or `$ISOLATION_STRING` where it doesn't populate the `confDir` always, but only populates it if the `--multus-conf-dir` is used.

Reply:
That makes more sense. I would agree.